3.0.4/postgres/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM geonetwork:3.0.4
 
 RUN apt-get update && apt-get install -y postgresql-client && \
-    apt-get clean && rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/*
```

Comment:
`apt-get clean` is performed automatically by the base image (see https://github.com/docker/docker/blob/9c2f1669a0b556115f543a0f02bf1ea5576b7f71/contrib/mkimage/debootstrap#L82-L105)